src/test/java/com/exasol/adapter/dialects/oracle/OracleVirtualSchemaIntegrationTestSetup.java

```diff
@@ -46,7 +47,8 @@ public class OracleVirtualSchemaIntegrationTestSetup implements Closeable {
         try {
             this.exasolContainer.start();
             this.oracleContainer.start();
-            uploadOracleJDBCDriverAndVSToBucket(this.exasolContainer.getDefaultBucket());
```

Comment:
This sounds very similar to `OracleSqlDialectIT.java`. 
Would it make sense to create a common setup class or method?

Reply:
We could also move this to a separate ticket: Refactor integration tests.

Reply:
We will move this to a separate ticket: Refactor integration tests.